packages/ifpack2/src/Ifpack2_Parameters.cpp

```diff
@@ -64,6 +67,17 @@ void getValidParameters(Teuchos::ParameterList& params)
   params.set("fact: iluk level-of-fill", (int)1);
   params.set("fact: iluk level-of-overlap", (int)0);
 
+  // Ifpack2_Chebyshev
+  params.set("chebyshev: max eigenvalue",STS::nan());
+  params.set("chebyshev: ratio eigenvalue",STS::nan());
+  params.set("chebyshev: min eigenvalue",(double)30.0);
+  params.set("chebyshev: degree",(int)1);
+  params.set("chebyshev: eigenvalue max iterations",(int)10);
+  params.set("chebyshev: assume matrix does not change",false);
+  // params.set("chebyshev: operator inv diagonal",Teuchos::null);
+  params.set("chebyshev: min diagonal value",STS::eps());
```

Comment:
Is the "min diagonal value" a magnitude (as `eps()` is) or a value?  That is, if `Scalar` is complex, should the min diagonal value be complex?

Reply:
The file assumes `Scalar=double`, so that's ok here.